src/App.tsx

```
@@ -1,12 +1,15 @@
 import Everytime from "@/components/Everytime";
 import LoginPage from "@/components/LoginPage";
+import Payment from "@/components/Payment";
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)